tracing-attributes/src/attr.rs

```diff
@@ -86,6 +86,8 @@ impl Parse for InstrumentArgs {
                 // XXX: apparently we support names as either named args with an
                 // sign, _or_ as unnamed string literals. That's weird, but
                 // changing it is apparently breaking.
+                // This also means that when using idents for name, it must be via
+                // a named arg.
```

Comment:
To make this comment a bit clearer for the next person who is reading it, it would be great if you included an example of what you mean.

Reply:
I'm sorry but I don't understand myself. I have some vague idea, but that's about it.

I just really need this feature, and I noticed that there's already a PR by someone else lingering, so I just picked it up, refreshed and making sure it lands.

Reply:
Oh, looks like the original author @mbrobbel is around. If anyone can paste the string I'm supposed to add there, I'm happy to do so and push the result. :D

Reply:
It uses a different parser when there is no named arg (`name =`), so you have to use `name = ` if you want to use an identifier of a const value for name.

Reply:
```
#[instrument(name = SOME_IDENT)]
```

instead of 

```
#[instrument(SOME_IDENT)]
```

?